ogr/services/github/project.py

```diff
@@ -373,6 +373,26 @@ def get_file_content(self, path: str, ref="master") -> str:
         except UnknownObjectException as ex:
             raise FileNotFoundError(f"File '{path}' on {ref} not found", ex)
 
+    def get_files(self, ref: str = "master") -> List[str]:
+        """
+        Get files of the project.
+
+        :return: [str]
+        """
+        files = []
+        contents = self.github_repo.get_contents(path="", ref=ref)
+
+        while contents:
+            file_content = contents.pop(0)
+            if file_content.type == "dir":
+                contents.extend(
+                    self.github_repo.get_contents(path=file_content.path, ref=ref)
```

Comment:
I am not sure about the pygithub/python-gitlab internals, but I want to be sure that we are not downloading the file contents. (If it is doable.)